packages/curve-ui-kit/src/widgets/Header/MobileHeader.tsx

```diff
@@ -83,13 +82,22 @@ export const MobileHeader = ({
           anchor="left"
           onClose={closeSidebar}
           open={isSidebarOpen}
-          slotProps={{ paper: { sx: { top, ...MOBILE_SIDEBAR_WIDTH, ...HIDE_SCROLLBAR } } }}
+          slotProps={{
+            paper: {
+              sx: {
+                top,
+                ...MOBILE_SIDEBAR_WIDTH,
+                ...HIDE_SCROLLBAR,
+                height: `calc(100dvh - ${top}px)`,
+              },
+            },
+          }}
           sx={{ top }}
           variant="temporary"
           hideBackdrop
           data-testid="mobile-drawer"
         >
-          <Box>
+          <Stack sx={{ overflowY: 'auto', ...HIDE_SCROLLBAR }}>
```

Comment:
The paper component has already overflow, and so does the drawer. Why do we need another one?

Reply:
Yeah, I removed it initially at first, but notice how the footer sits outside of this stack. That footer must not be in the scrollable area.